scripts/coe_stats_gui.lua

```diff
@@ -0,0 +1,105 @@
+---@class coe.StatsGUI
+local StatsGUI = {}
+
+local mod_gui = require( "mod-gui" )
+
+-- =============================================================================
+
+local function build_stats_info_frame( statistics_frame )
+  local info_frame = statistics_frame.add{
+    type = "frame",
+    name = "coe_stats_info_frame",
+    direction = "vertical"
+  }
+
+  info_frame.add{
+    type = "label",
+    caption = {"coe-stats-gui.label_launches_title"}
+  }.parent.add{
+    type = "flow",
+    direction = "horizontal"
+  }.add{
+    type = "label",
+    caption = {"coe-stats-gui.label_launches_required"}
+  }.parent.add{
+    type = "label",
+    caption = tostring( global.rocket_silo.required_launches )
+  }.parent.parent.add{
+    type = "flow",
+    direction = "horizontal"
+  }.add{
+    type = "label",
+    caption = {"coe-stats-gui.label_launches_complete"}
+  }.parent.add{
+    type = "label",
+    caption = tostring( global.rocket_silo.total_launches )
+  }
+end -- build_stats_info_frame
+
+-------------------------------------------------------------------------------
+
+local function open_stats_ui(event)
+  local player = game.get_player(event.player_index)
+  local gui = player.gui.left
```

Comment:
Statistics frame should open under the mod button

Reply:
I do not understand this request.